x-pack/plugins/reporting/public/components/report_listing.tsx

```diff
@@ -143,30 +160,50 @@ export class ReportListing extends Component<Props, State> {
       },
       {
         field: 'status',
-        name: 'Status',
+        name: intl.formatMessage({
+          id: 'xpack.reporting.listing.tableColumns.statusTitle',
+          defaultMessage: 'Status',
+        }),
         render: (status: string, record: Job) => {
-          let maxSizeReached;
-          if (record.max_size_reached) {
-            maxSizeReached = <span> - max size reached</span>;
-          }
           let statusTimestamp;
-          if (status === 'processing' && record.started_at) {
+          if (status === JobStatuses.JOB_STATUS_PROCESSING && record.started_at) {
```

Comment:
I believe JobStatuses.JOB_STATUS_PROCESSING => jobStatuses.PROCESSING. The same for JobStatuses.JOB_STATUS_COMPLETED and other statuses. Also JobStatuses should start with lowercase